analysis_module/views.py

```diff
@@ -16,18 +16,18 @@
 
 from .utils import spin_ecs_container
 from .models import AnalysisModuleRequest
-from .mockserver import topicmodelingmodel, ngramsmodel, summarizationmodel, geolocationmodel
+from .mockserver import topicmodeling_model, ngrams_model, summarization_model, geolocation_model
 
 
 def process_mock_request(request: dict, type: str):
     if type == "topicmodel":
-        response, code = topicmodelingmodel(request)
+        response, code = topicmodeling_model(request)
     elif type == "summarization":
-        response, code = summarizationmodel(request)
+        response, code = summarization_model(request)
     elif type == "ngrams":
-        response, code = ngramsmodel(request)
+        response, code = ngrams_model(request)
     elif type == "geolocation":
-        response, code = geolocationmodel(request)
+        response, code = geolocation_model(request)
```

Comment:
Can we do something like by avoiding if .. else....

action = {
    "topicmodel": topicmodeling_model,
    "summarization": summarization_model,
    "ngrams": ngrams_model,
    "geolocation": geolocation_model
    }
    
action.get(type)(request)

Something like this.

Reply:
Done.